app/templating/view_context.py

```diff
@@ -98,14 +98,15 @@ def build_view_context_for_question(metadata, schema, answer_store, current_loca
 
 
 def build_view_context_for_final_summary(metadata, schema, answer_store, schema_context,
-                                         block_type, variables, csrf_token):
+                                         block_type, variables, csrf_token, rendered_block):
     section_list = renderer.render(schema.json, **schema_context)['sections']
 
     context = build_view_context_for_summary(schema, section_list, answer_store, metadata,
                                              csrf_token, block_type, variables)
 
     context['summary'].update({
         'is_view_submission_response_enabled': is_view_submitted_response_enabled(schema.json),
+        'collapsible': _get_collapsible_final_summary(rendered_block),
```

Comment:
Do we really need a separate method for this?

`'collapsible': rendered_block.get('collapsible', False),`

Reply:
Also strictly speaking we don't need to pass `block_type` now as we can get that from the `rendered_block['type']`.